src/containers/app/App.test.js

```diff
@@ -0,0 +1,20 @@
+<<<<<<< HEAD
+import React from 'react';
+<<<<<<< HEAD:src/App.test.js
+import { shallow } from 'enzyme';
+import App from './App';
+=======
```

Comment:
I think you need to check on this merge conflict statements that are causing tests to fail

Reply:
Thanks for pointing this out, am working on it.